manual/en-US/coding-standards/chapters/php.md

```diff
@@ -131,6 +131,25 @@ switch ($value)
 }
 ```
 
+## Mixed language usage (e.g. at the layout files)
+
+For layout files and all files where we use a mix of PHP and HTML (all PHP files in the `view/tmpl` and `layout` folder) we wrapped additionally every line into a `<?php ... ?>` block and use the alternative syntax for control structures.
+Reason is that this way is supposed to be easier to understand by frontend guys and regular users. You can also easier move blocks around without creating fatal errors due to missing `<?php ... ?>` tags and the like.
```

Comment:
I would try to find a better way to word this.  I've read this sentence a couple of times today and each time the first thing that I react with is "man this feels like we're talking down on non-developers".